airlock/issues.py

```diff
@@ -204,3 +186,39 @@ def update_output_checking_issue(
         )
 
     return comment_url
+
+
+def update_output_checking_issue(
+    release_request_id,
+    workspace_name,
+    updates,
+    org,
+    repo,
+    github_api,
+    notify_slack,
+    request_author,
```

Comment:
It would be nice not to have to pass around all these arguments (especially as they're positional args, so order matters).  

Probably out of scope for this ticket, but I think we could reuse the AirlockEvent object. It has most of the things that we pass to the output checking issues. And updates could be fetched in the issues functions instead of being called first in views.py.